auth_signup_verify_email/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Verify email at signup
+======================
+
+This module was written to extend the functionality of public sign up to force
+users to provide a valid email address.
+
+To reinforce this requirement, the user will not need to provide a password at
+sign up, but when logging in later for the first time.
+
+Installation
+============
+
+* Install validate_email_ with ``pip install validate_email`` or equivalent.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* `Properly configure your outgoing email server(s)
+  <https://www.odoo.com/es_ES/forum/help-1/question/how-to-configure-email-gateway-282#answer_290>`_.
+* Go to *Settings > Configuration > General Settings* and enable *Allow
+  external users to sign up*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Log out.
+* `Sign up </web/signup>`_ with a valid email.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/8.0
```

Comment:
This is now repo 149 ;-) https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt#L28